ampeg.testing/src/test/java/ProductLinkTest.java

```diff
@@ -0,0 +1,73 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ProductLinkTest extends AmpegTest {
+	@Test
+	public void ensureArtistOnProductDetailsPage() {
+		
+		String expected = "Bootsy Collins";
+
+		String actualName = new HomePage(this.driver)
+				.openProductPage()
+				.openClassicSeriesBassHeads()
+				.openSVT410HLF()
+				.getArtist();
+		;
+
+		Assert.assertEquals(actualName, expected, "Test should find artist name");
+	}
+
+	@Test
+	public void downloadUserManual() {
+		
+		String expectedURL = "https://ampeg.com/data/6/0a020a40dc385e0b9cc618bd0/application/pdf/Heritage%2050th%20Anniversary%20SVT%20Quick%20Start%20Guide%20-%20English%20.pdf";
```

Comment:
0a020a40dc385e0b9cc618bd0
This appears to be a dynamic value. If so, how often will it change?

Reply:
changed test to only look at file name